packages/storefront-sdk/src/client/index.ts

```diff
@@ -30,26 +35,70 @@ export interface QueryParams<QData, QVariables extends AnyVariables> {
 }
 
 export class StorefrontClient {
-	readonly #graphqlClient: UrqlClient;
+	#graphqlClient: UrqlClient;
 	readonly #afterSubscriptions: AfterSubscriptions<DataFetchingMethodName>;
+	#storefrontEndpoint: string;
+	#previewToken: string | undefined;
+	#locale: string;
+	#fetchClient: typeof globalThis.fetch;
```

Comment:
Would it make sense to wrap these into a `#config` property? Just for logical grouping purposes?